easybuild/easyconfigs/z/zeus-mcmc/zeus-mcmc-2.5.4-foss-2022a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'zeus-mcmc'
+version = '2.5.4'
+
+homepage = 'https://zeus-mcmc.readthedocs.io'
+description = "Zeus is a Python implementation of the Ensemble Slice Sampling method."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-learn', '1.1.2'),
+    ('tqdm', '4.64.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('seaborn', '0.12.2', {
+        'checksums': ['374645f36509d0dcab895cba5b47daf0586f77bfe3b36c97c607db7da5be0139'],
+    }),
+    (name, version, {
+        'modulename': 'zeus',
+        'checksums': ['594baa90de4ad4488c4db5ed6a0446f7103bc4b3de787f4d7d23c91c9aa88769'],
+    }),
+]
+
+moduleclass = 'lib'
```

Comment:
well, `lib` is arguably better than `emcee`s `astro`, but since this is the first easyconfig for `zeus-mcmc`, how do you feel about changing this to `math`?

Reply:
both packages look completely generic MCMC implementations to me, I don't see anything domain specific about them

but indeed `moduleclass` is often subjective and nothing is enforced at all, so definitely not a hill worth dying on :)

Reply:
change `lib` → `math` done.  ✅